templates/ingress-metadata.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  labels:
+    k8s-app: metadata-service
+  name: metadata-ingress
+  annotations:
+    nginx.ingress.kubernetes.io/ssl-redirect: "false"
+    nginx.ingress.kubernetes.io/use-regex: "true"
+spec:
+  ingressClassName: nginx-static
+  rules:
+    - host: metadata-service.packet.net
```

Comment:
Same thing, the hosts should be in `values.yaml` and range over them here?

Reply:
It seemed like there was a lot going on in the serverservice helm chart to do this (flags for whether clusterURI was enabled, splitting out subdomains, etc), so I kept it simple and just moved the list definition to values.yaml and iterated on it. Let me know if keeping it simple was the right approach here.